src/shared/devnode-acl.c

```diff
@@ -195,21 +195,12 @@ int devnode_acl_all(const char *seat,
         dir = opendir("/run/udev/static_node-tags/uaccess");
         if (dir) {
                 FOREACH_DIRENT(de, dir, return -errno) {
-                        _cleanup_free_ char *unescaped_devname = NULL;
-                        ssize_t l;
-
-                        l = cunescape(de->d_name, UNESCAPE_RELAX, &unescaped_devname);
-                        if (l < 0)
-                                return l;
-
-                        n = path_join("/dev", unescaped_devname);
-                        if (!n)
-                                return -ENOMEM;
+                        r = readlinkat_malloc(dirfd(dir), de->d_name, &n);
```

Comment:
maybe skip ENOENT here, in case this code runs in paralell with udev's logic and an entry that was still in the directory moments ago is not anymore by the time we try to read it.

In fact, I'd probably do it like this: 

```c
r = readlink_malloc(dirfd(dir), de->d_name, &n);
if (r == -ENOENT)
        continue;
if (r < 0) {
        log_debug_errno("Unable to read symlink '/run/udev/static_node-tags/uaccess/%s', ignoring: %m", de->d_name);
        continue;
}
```

i.e. totally ignore ENOENT, and log but ignore all other errors.